src/MaaWpfGui/Res/Localizations/en-us.xaml

```diff
@@ -615,6 +615,7 @@ The video aspect ratio needs to be 16:9 without interference factors such as bla
     <system:String x:Key="EmergencyDps">Node: Emergency Operation</system:String>
     <system:String x:Key="DreadfulFoe">Node: Dreadful Foe</system:String>
     <system:String x:Key="RoguelikeEvent">Event:</system:String>
+    <system:String x:Key="RoguelikeInvestment">Invested {1}(+{0}), deposit: {2}</system:String>
```

Comment:
deposit is the current total amount of already invested ingots correct?
en i18n lgtm

Reply:
yes, it's an ocr result from here.
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/102887808/1a043099-a02a-4273-8ce8-8c5d42c82e03)